spec/javascripts/faye-extension_spec.js

```diff
@@ -71,7 +74,6 @@ describe('Faye extension', function() {
         this.dispatcher = {connectionType: "fake", clientId: '1234', sendMessage: function() {}, selectTransport: function() { }};
         spyOn(this.dispatcher, 'sendMessage');
         spyOn(this.dispatcher, 'selectTransport');
-        Faye.extend(this.dispatcher, Faye.Publisher)
```

Comment:
got a Faye.extend is not a function error, fixed by removing the line, at this point I've no idea what I'm doing